docs/howto/kubernetes/gardener/kubectl.md

```diff
@@ -66,9 +46,9 @@ Create a directory named `.kube` in your local user's home, then move the YAML f
 Rename the YAML to `config`, ending up with `~/.kube/config`.
 By default, `kubectl` searches for its configuration in `~/.kube/config`.
 If necessary, you can [modify this behavior](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) by setting the `KUBECONFIG` environment variable.
-For example, if you want to retain the original filename exactly as downloaded, you will type something like the following:
+For example, if you want to retain the original filename exactly as downloaded, you might type something like the following:
 
-```shell
+```console
```

Comment:
`shell` is for highlighting shell commands. `console` is for console screen dumps.

So, either leave `shell`, or else use `console` and add a console prefix like `$`.

I'd prefer to keep `shell`.

Reply:
I am switching to `shell`. By thw way, first I tried `console` with the `$` prefix, but `tox -e markdownlint` doesn't like the fact there's no command output, so it aborts like so: 

```
MD014: Dollar signs used before commands without showing output (commands-show-output)
```